src/helpers.js

```diff
@@ -1,6 +1,6 @@
 const p = require('path')
-const babel = require('babel-core')
-const template = require('babel-template')
+const babel = require('@babel/core')
```

Comment:
You're already passing in `transformFromAst` in https://github.com/kentcdodds/babel-plugin-codegen/blob/b41bb53428b8687de5b279e79349222cb8c68ea5/src/index.js#L6, is it an option to pass in `transform` and `template` too? They are also all available on that options object, and you'd avoid the dependency on `babel-core` entirely.